cli/README.md

```diff
@@ -77,7 +77,7 @@ cargo run --bin kagami -- crypto --json
 
 For the Iroha peer binary to run, a configuration file must be provided. Iroha will not run with defaults if the configuration file is not available.
 
-The Iroha binary looks for either a file `config.json` in the current directory, or for a JSON file `IROHA2_CONFIG_PATH`. If the latter environment variable is defined, but not a valid configuration file, the Iroha peer binary will exit and do nothing.
+The Iroha binary looks for either a file `config.json` in the current directory, or for a JSON file `IROHA2_CONFIG_PATH`. If the latter environment variable is defined, but not a valid configuration file, the Iroha peer binary will exit and do nothing. If neither of those is provided, all the fields from default `config.json` should be specified as environment variables.
```

Comment:
As I see from the implementation it seems that env variable `IROHA2_CONFIG_PATH` takes precedence over `config.json` in current directory

Reply:
```suggestion
The Iroha binary looks for either a file `config.json` in the current directory, or for a JSON file `IROHA2_CONFIG_PATH`. If the configuration file is not valid, the Iroha peer binary will exit and do nothing. If neither of those is provided, all the fields from default `config.json` should be specified as environment variables.
```